rpm/util-linux.spec

```diff
@@ -332,7 +312,7 @@ done
 %pre -n uuidd
 getent group uuidd >/dev/null || groupadd -r uuidd
 getent passwd uuidd >/dev/null || \
-useradd -r -g uuidd -d /var/lib/libuuid -s /sbin/nologin \
+useradd -r -g uuidd -d /var/lib/libuuid -s %{_sbindir}/nologin \
```

Comment:
If `uuidd` package was previously installed, user uuidd was already created with `/sbin/nologin` as shell.  In that case we should probably change the shell to point to `/usr/sbin/nologin` here. Then again, I don't think we actually use `uuidd` anywhere or do we? Could we perhaps just get rid of the package?

Reply:
Good point. I added a symlink before we have a symlink from `/usr/bin` to `/bin` after we have fully done the UsrMove.